example-crates/wallet_rpc/README.md

```diff
@@ -14,14 +14,17 @@ ARGS:
     <CHANGE_DESCRIPTOR>    Wallet change descriptor [env: CHANGE_DESCRIPTOR=]
 
 OPTIONS:
-        --db-path <DB_PATH>
-            Where to store wallet data [env: BDK_DB_PATH=] [default: .bdk_wallet_rpc_example.db]
-
+    --change-descriptor <CHANGE_DESCRIPTOR>
+            Wallet change descriptor [env: CHANGE_DESCRIPTOR=] [default:
+            wpkh(tprv8ZgxMBicQKsPdy6LMhUtFHAgpocR8GC6QmwMSFpZs7h6Eziw3SpThFfczTDh5rW2krkqffa11UpX3XkeTTB2FvzZKWXqPY54Y6Rq4AQ5R8L/84'/1'/0'/1/*)]
+    --descriptor <DESCRIPTOR>
+            Wallet descriptor [env: DESCRIPTOR=] [default:
+            wpkh(tprv8ZgxMBicQKsPdy6LMhUtFHAgpocR8GC6QmwMSFpZs7h6Eziw3SpThFfczTDh5rW2krkqffa11UpX3XkeTTB2FvzZKWXqPY54Y6Rq4AQ5R8L/84'/0'/0'/1/*)]
```

Comment:
@storopoli I'm unsure if we should keep the default info here, as it could be misleading, or add an example below with these descriptors, wdyt?

Reply:
Yes I've forgot those to change it. will do. nice catch.

Reply:
Done!

Reply:
Awesome, thanks!